.github/workflows/auto_cherry_pick.yml

```diff
@@ -0,0 +1,29 @@
+name: Auto Cherry Pick 
+on:
+  workflow_dispatch:
+    inputs:
+      pr_label:
+        description: "Cherry pick label such as 'cherry-pick-v3.2'"
+        required: true
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  auto-cherry-pick:
+      #if: ${{ startsWith(github.event.pull_request.labels.*.name, 'cherry-pick-') && github.event.pull_request.merged == true }}
+      runs-on: ubuntu-latest
+      container:
+        image: reg.vesoft-inc.com/dashboard/dashboard-dev:centos7
```

Comment:
Why don't you run in `ubuntu-latest` directly?

Reply:
nened git version and node version